app/services/search/algolia_search.rb

```diff
@@ -0,0 +1,50 @@
+class Search::AlgoliaSearch
+  attr_reader :vacancies, :stats
+
+  def initialize(search_params)
+    @keyword = search_params[:keyword]
+    @coordinates = search_params[:coordinates]
+    @radius = search_params[:radius]
+    @polygon = search_params[:polygon]
+    @filters = search_params[:filters]
+    @replica = search_params[:replica]
+    @hits_per_page = search_params[:hits_per_page]
+    @page = search_params[:page]
+    @typo_tolerance = search_params[:typo_tolerance]
```

Comment:
I don't understand: in what scenario is it possible for @typo_tolerance to be not nil?
- It's not from searching because that's not a query param the user can set
- It's not from AlertBuilder because that overwrites this initialize method (and uses 'false' hard-coded anyway)

Reply:
It's from AlertBuilder. AlertBuilder < SearchBuilder, this is AlgoliaSearch

Reply:
That seems to validate this comment:

> I foresee some confusion in my future about whether I need to look in search/algolia_search.rb or search/search_builder.rb

Reply:
I think my confusion is mainly seeing the word 'search' and thinking 'ah that's exactly the same, no need to read the rest of THIS file name!'

Reply:
Perhaps `AlgoliaSearchRequest` might make it clearer, if people are happy with that